app/poros/i_scream_results.rb

```diff
@@ -5,13 +5,33 @@ def service
   end
 
   def game_params(game_name)
+    game_name = game_name.gsub(" ", "%20")
     service.get_game_exact(game_name)
   end
 
+  def existing_game(game_name)
+    Game.find_by(name: game_name)
```

Comment:
Dont hit the API if the game is already in the DB. kinda feels like caching should replace this